business_register/converter/uo_historical.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+from django.apps import apps
+
+from business_register.models.company_models import Company
+from data_converter import settings_local
+from data_ocean.converter import Converter
+
+
+class Historical(Converter):
+    LOCAL_FILE_NAME = settings_local.LOCAL_FILE_NAME_UO_ADDRESS
+    LOCAL_FOLDER = settings_local.LOCAL_FOLDER
```

Comment:
use django interface for access to settings
```suggestion
from django.conf import settings
from data_ocean.converter import Converter


class Historical(Converter):
    LOCAL_FILE_NAME = settings.LOCAL_FILE_NAME_UO_ADDRESS
    LOCAL_FOLDER = settings.LOCAL_FOLDER
```

Reply:
done

Reply:
fixed